src/components/AccountNotifications.tsx

```diff
@@ -14,10 +13,21 @@ interface IProps {
 export const AccountNotifications = (props: IProps) => {
   const { hostname, showAccountHostname, notifications } = props;
 
-  const groupedNotifications = _(notifications)
-    .groupBy((obj) => obj.repository.full_name)
-    .sortBy((_, key) => key)
-    .value();
+  const groupedNotifications = Object.values(
+    notifications.reduce(
```

Comment:
FWIW, my eyes glaze over with this, where the previous code was understandable at a glance.

Any way to maintain the readability?

Reply:
Not a blocker